RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromProtoTracksEDProducer.cc

```diff
@@ -24,6 +24,7 @@
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include <vector>
+#include <cassert>
```

Comment:
For 74X there is no strict need to remove the assert, as the code causing it to fire is only in 75X. On the other hand, removing it from 74X should not cause any problems either.